docs/how-to-guides/meson-args.rst

```diff
@@ -4,77 +4,54 @@
 
 .. _how-to-guides-meson-args:
 
-***************************
-Passing arguments to Meson_
-***************************
+**************************
+Passing arguments to Meson
+**************************
```

Comment:
Why?

Reply:
Because it looks nicer and we don't need to make every mention to Meson a link to the Meson home page. We don't do it in the text and I don't see any reason to do it in titles.